yb-voyager/src/tgtdb/conn_pool.go

```diff
@@ -280,6 +281,32 @@ func (pool *ConnectionPool) shuffledConnUriList() []string {
 	return connUriList
 }
 
+func (pool *ConnectionPool) RemoveConnectionsForHosts(servers []string) error {
+	log.Infof("Checking for connections on host: %s", servers)
+	var conn *pgx.Conn
+	var gotIt bool
+	size := pool.size
+	for i := 0; i < size; i++ {
+		conn, gotIt = <-pool.conns
+		if !gotIt {
+			break
```

Comment:
why break here?

Reply:
As per my understanding, I think this gotIt will be false in case the pool doesn't have connections or all conns are busy so in such case, it doesn't make sense to go further we keep on getting gotIT false only I think ?🤔

Reply:
I think you're right. 
Let's discuss this on call. I also want to go through the existing implementation of the conn pool to see if we might be missing anything.